gramps_webapi/api/__init__.py

```diff
@@ -74,9 +79,55 @@ def register_endpt(resource: Type[Resource], url: str, name: str):
 register_endpt(TranslationsResource, "/translations/", "translations")
 # Relation
 register_endpt(
-    RelationResource,
-    "/relations/<string:handle1>/<string:handle2>",
-    "relations",
+    RelationResource, "/relations/<string:handle1>/<string:handle2>", "relations",
 )
 # Metadata
 register_endpt(MetadataResource, "/metadata/<string:datatype>", "metadata")
+
+# Media files
+@api_blueprint.route("/media/<string:handle>/file")
+@jwt_required_ifauth
+def download_file(handle):
+    """Download a file."""
+    base_dir = current_app.config["MEDIA_BASE_DIR"]
```

Comment:
I know Gramps has get_mediapath/set_mediapath methods to store it in the database. How does that related to having the MEDIA_BASE_DIR config option?

Reply:
Oh, future S3 support I guess. Maybe I just answered my own question.